Makefile

```diff
@@ -6,35 +6,42 @@ NAMESPACE ?= nmstate-default
 IMAGE_REGISTRY ?= quay.io/nmstate
 IMAGE_TAG ?= latest
 PULL_POLICY ?= Always
-STATE_HANDLER_IMAGE ?= kubernetes-nmstate-state-handler
-POLICY_HANDLER_IMAGE ?= kubernetes-nmstate-configuration-policy-handler
+OPERATOR_IMAGE ?= kubernetes-nmstate
+NMSTATECTL_IMAGE ?= nmstatectl
```

Comment:
I don't want to conflict with whatever nmstate/nmstate people are doing. What if we find out that we need some custom kubernetes-nmstate code in the DS part? Then it won't be a generic "nmstatectl" image anymore. I blab more about the naming in following comment.

Reply:
For images:
- kubernetes-nmstate-manager
- kubernetes-nmstate-handler

For objects (on kube API, namespace/name):
- deployment kubernetes-nmstate/manager
- DS kubernetes-nmstate/handler